### PR TITLE
Pellet

### PR DESCRIPTION
**Pull request by knuch**
Added a "Pellet" component to display radial links. 
A JS scans the height of the text inside the pellet and adapts font-size/line-height if too big

### CODE REVIEW

File: assets/sass/atoms/_pellet.scss [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: ($spacer * 12);
+  height: ($spacer * 12);
+  background: $deep-blue;
+  border-radius: ($spacer * 6);
+  transform: rotate(-15deg);
+  box-shadow: 0 0 29px -5px rgba(0, 0, 0, 0.93);
+  transition: 0.2s ease;
```
Comment (tonifisler) [inline assets/sass/atoms/_pellet.scss]: Specify a propriety for the transition

File: assets/components/atoms/pellet.html.swig [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    - `.bg-highlight` – magenta background
+    - `.bg-pellet-white` – white background
+    - `.text-secondary` – cyan text
+    - `.text-highlight` – magenta text
+---
+
+<a href="#" class="pellet
+{%- if variant == 'pellet-secondary' %} bg-secondary{% endif -%}
+{%- if variant == 'pellet-highlight' %} bg-highlight{% endif -%}
+{%- if variant == 'pellet-negative' %} bg-pellet-white{% endif -%}
```
Comment (tonifisler) [inline assets/components/atoms/pellet.html.swig]: This line doesn't seem usefull

File: assets/sass/atoms/_pellet.scss [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+  &:hover p {
+    color: $light-blue;
+  }
+}
+
+// variants
+.pellet {
+
+  //negative colors
+  &.bg-pellet-white {
```
Comment (tonifisler) [inline assets/sass/atoms/_pellet.scss]: `.pellet-bg-white` seems more logical to me

File: assets/sass/atoms/_pellet.scss [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+    p {
+      color: $deep-blue;
+    }
+
+    //negative hover styles
+    &:hover p {
+      color: $light-blue;
+
+      &.text-secondary {
+        color: $light-blue-dark !important;
```
Comment (tonifisler) [inline assets/sass/atoms/_pellet.scss]: Explain why the `!important` in a comment :)

Comment (knuch) [inline assets/sass/atoms/_pellet.scss]: .text-secondary has an !important statement. To overwrite it and get the desired hover color effet, this was necessary

**Pull request merged**